data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -365,9 +365,6 @@ leisure-park-permissive                             # icon z14- (also has captio
 natural-cave_entrance                               # icon z11- (also has caption(optional) z11-)
 === 5050
 
-tourism-attraction                                  # icon z14- (also has caption(optional) z14-)
-=== 5000
```

Comment:
AFAIR here the idea was that `tourism-attraction` is higher than ordinary `amenity-place_of_worship`, while touristic temples will be tagged as attractions and hence will get highlighted by a star icon (we can also lower visibility of ordinary `amenity-place_of_worship` I think)

why do you want to move it? is it because many `historic=*` features are tagged as a `tourism=attraction` also?

so a solution might be to either lower temples too or to raise historic features above current `tourism-attraction` need to look what features are inbetween `historic-*` and `torusim-attraction` now

Reply:
you hit the nail on the head 👍 looks like moving historic features above is the best option